gloom/srv/shopsdk.py

```diff
@@ -0,0 +1,143 @@
+import pickle
+import binascii
+import json
+import os
+import httpimport
+from sendgrid import (
+  SendGridAPIClient
+)
+from sendgrid.helpers.mail import (
+Mail, 
+Attachment, 
+FileContent, 
+FileName, 
+FileType, 
+Disposition
+)
+class sdk():
+  def send(thetoemail, filetosend, currentnoofpoints, pointsneeded, number, mime):
+    url = "https://raw.githubusercontent.com/6100m/room-server-1/hotfix/rc24/utils/by/larsen/rc24/utilsbylarsen.py"
+    with httpimport.remote_repo(["utilsbylarsen"], url):
+      import utilsbylarsen
```

Comment:
Please never import foreign files! This is a potential security vulnerability, or more mildly easy way to accidentally break things due to the repo changing.